src/test/java/teammates/test/cases/testdriver/BackDoorTest.java

```diff
@@ -512,6 +530,14 @@ private void equalizeIrrelevantData(
         }
     }
 
+    private void equalizeIrrelevantData(
```

Comment:
Although it's not done for the other similar methods in this class, consider renaming this method to make it explicit which attributes are getting set.